[[!comment format=mdwn
 username="https://www.google.com/accounts/o8/id?id=AItOawltxdgYMUK4CMJh3jC8AlegwyoiHA9Ka7o"
 nickname="Justin"
 subject="comment 5"
 date="2014-01-29T20:26:15Z"
 content="""
This seems to have affected a lot of tests, I'm now getting \"20 out of 72 tests failed\".  In particular, error messages similar to the following show up now in many tests (with different paths for each test).

    git-annex: MoveFile \"c:\\Users\\geibeljt\\annex\\.t\\tmprepo36\\.git\\annex\\objects\\574\\4ca\\SHA256E-s7--6f
    e7bd6dcd1d46cfa98027c64f1e51d59664989c3b1e80a980bac1cb352a2ba7\\SHA256E-s7--6fe7bd6dcd1d46cfa98027c64f1e51d596
    64989c3b1e80a980bac1cb352a2ba7.map25112.tmp\" \"c:\\Users\\geibeljt\\annex\\.t\\tmprepo36\\.git\\annex\\objects\
    \574\\4ca\\SHA256E-s7--6fe7bd6dcd1d46cfa98027c64f1e51d59664989c3b1e80a980bac1cb352a2ba7\\SHA256E-s7--6fe7bd6dc
    d1d46cfa98027c64f1e51d59664989c3b1e80a980bac1cb352a2ba7.map\": already exists (Cannot create a file when that f
    ile already exists.)


---
And here is the full output for the test that was originally failing.  It now appears to fail much earlier than before.

        conflict_resolution (mixed directory and file):   Detected a filesystem without fifo support.
      Disabling ssh connection caching.
      Detected a crippled filesystem.
      Enabling direct mode.
      Detected a filesystem without fifo support.
      Disabling ssh connection caching.
      Detected a crippled filesystem.
      Enabling direct mode.
    add conflictor ok
    (Recording state in git...)
    (merging origin/git-annex origin/synced/git-annex into git-annex...)
    (Recording state in git...)
    commit  ok
    pull origin
    ok
    push origin
    Counting objects: 21, done.
    Delta compression using up to 4 threads.
    Compressing objects: 100% (13/13), done.
    Writing objects: 100% (16/16), 1.57 KiB | 0 bytes/s, done.
    Total 16 (delta 3), reused 0 (delta 0)
    To c:/Users/geibeljt/annex/.t\repo
       a462941..43dcc49  git-annex -> synced/git-annex
       8268d47..3012c35  annex/direct/master -> synced/master
    ok
    add conflictor/subfile ok
    (Recording state in git...)
    (merging origin/git-annex origin/synced/git-annex into git-annex...)
    (Recording state in git...)
    commit
    git-annex: MoveFile \"c:\\Users\\geibeljt\\annex\\.t\\tmprepo36\\.git\\annex\\objects\\574\\4ca\\SHA256E-s7--6f
    e7bd6dcd1d46cfa98027c64f1e51d59664989c3b1e80a980bac1cb352a2ba7\\SHA256E-s7--6fe7bd6dcd1d46cfa98027c64f1e51d596
    64989c3b1e80a980bac1cb352a2ba7.map25112.tmp\" \"c:\\Users\\geibeljt\\annex\\.t\\tmprepo36\\.git\\annex\\objects\
    \574\\4ca\\SHA256E-s7--6fe7bd6dcd1d46cfa98027c64f1e51d59664989c3b1e80a980bac1cb352a2ba7\\SHA256E-s7--6fe7bd6dc
    d1d46cfa98027c64f1e51d59664989c3b1e80a980bac1cb352a2ba7.map\": already exists (Cannot create a file when that f
    ile already exists.)
    failed
    pull origin
    remote: Counting objects: 21, done.
    remote: Compressing objects: 100% (13/13), done.
    remote: Total 16 (delta 3), reused 0 (delta 0)
    Unpacking objects: 100% (16/16), done.
    From c:/Users/geibeljt/annex/.t\repo
       8268d47..3012c35  master     -> origin/master
       a462941..43dcc49  synced/git-annex -> origin/synced/git-annex
       8268d47..3012c35  synced/master -> origin/synced/master
    
    Updating 8268d47..3012c35
    Fast-forward
     conflictor | 1 +
     1 file changed, 1 insertion(+)
     create mode 120000 conflictor
    ok
    (merging origin/synced/git-annex into git-annex...)
    (Recording state in git...)
    push origin
    Counting objects: 24, done.
    Delta compression using up to 4 threads.
    Compressing objects: 100% (13/13), done.
    Writing objects: 100% (16/16), 1.42 KiB | 0 bytes/s, done.
    Total 16 (delta 6), reused 0 (delta 0)
    To c:/Users/geibeljt/annex/.t\repo
       43dcc49..c040cd6  git-annex -> synced/git-annex
    ok
    git-annex: sync: 1 failed
    FAIL
"""]]
